### PR TITLE
Facebook link and stuff

### PR DESCRIPTION
**Pull request by MoOx**
This PR:

- relocate some global css
- use css modules for navigation
- move global css variables in webpack config
- add a facebook link on french pages only (header/footer)
- change chat icon
- reduce icons size in header
- prevent icon tooltip from being below i18nBanner

Before

screen shot 2016-03-01 at 11 57 10
screen shot 2016-03-01 at 11 57 13

After

screen shot 2016-03-01 at 11 57 54
screen shot 2016-03-01 at 11 57 56

### CODE REVIEW

**Comment by MoOx** [comment]
Don't worry about failing tests. #647

File: web_modules/icons/chat.svg [hunk 1]
```diff
@@ -1,41 +1 @@
...
+<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24"><path d="M10 3.002c4.411 0 8 2.849 8 6.35 0 3.035-3.029 6.311-7.925 6.311-1.58 0-2.718-.317-3.718-.561-.966.593-1.256.813-3.006 1.373.415-1.518.362-2.182.331-3.184-.837-1.001-1.682-2.069-1.682-3.939 0-3.501 3.589-6.35 8-6.35zm0-2.002c-5.281 0-10 3.526-10 8.352 0 1.711.615 3.391 1.705 4.695.047 1.527-.851 3.718-1.661 5.312 2.168-.391 5.252-1.258 6.649-2.115 1.181.289 2.312.421 3.382.421 5.903 0 9.925-4.038 9.925-8.313 0-4.852-4.751-8.352-10-8.352zm11.535 11.174c-.161.488-.361.961-.601 1.416 1.677 1.262 2.257 3.226.464 5.365-.021.745-.049 1.049.138 1.865-.892-.307-.979-.392-1.665-.813-2.127.519-4.265.696-6.089-.855-.562.159-1.145.278-1.74.364 1.513 1.877 4.298 2.897 7.577 2.1.914.561 2.933 1.127 4.352 1.385-.53-1.045-1.117-2.479-1.088-3.479 1.755-2.098 1.543-5.436-1.348-7.348zm-15.035-3.763c-.591 0-1.071.479-1.071 1.071s.48 1.071 1.071 1.071 1.071-.479 1.071-1.071-.48-1.071-1.071-1.071zm3.5 0c-.591 0-1.071.479-1.071 1.071s.48 1.071 1.071 1.071 1.071-.479 1.071-1.071-.48-1.071-1.071-1.071zm3.5 0c-.591 0-1.071.479-1.071 1.071s.48 1.071 1.071 1.071 1.071-.479 1.071-1.071-.48-1.071-1.071-1.071z"/></svg>
```
Comment (magsout) [inline web_modules/icons/chat.svg]: Pourquoi compresser le SVG alors que le loader svgo le fait ? Autant garder l'original ?

Comment (MoOx) [inline web_modules/icons/chat.svg]: j'ai rien compressé, j'ai dl un nouveau icon sur iconmonstr, ils sont comme ça de base now.

Comment (magsout) [inline web_modules/icons/chat.svg]: Ah ok, au temps pour moi alors

**Comment by Nyalab** [comment]
you made me review way more CSS than i can for a day :laughing: 

lgtm

**Comment by magsout** [comment]
Nice

:shipit:

**Comment by MoOx** [comment]
Thank you for the contribution. It was merged directly as 085c92c85a3f7a4d0a09cb262767201dc9cdc5af to keep commit history cleaner. Your contribution is still credited to you.

**Pull request closed without merging**